workspaces/diff-engine/package.json

```diff
@@ -18,6 +18,7 @@
   "types": "lib/index.d.ts",
   "dependencies": {
     "@iarna/toml": "^2.2.5",
+    "@useoptic/diff-engine-wasm": "9.0.11-beta.0",
```

Comment:
not expecting it to change as part of this pr, but we should fix the dependencies here when we reorganize the packages too